### PR TITLE
Development footer and about page

### PR DESCRIPTION
**Pull request by jwu910**
Finishing up work on Footer and added temp link on About us page.

@RebWu 's next issue should redesign the Footer and will revisit then.

### CODE REVIEW

**Comment by jwu910** [comment]
[4b011e9](https://github.com/5-gwoap/adopt-a-family/pull/37/commits/4b011e9239cce86d398065db6d68e79f5e08ea61) fixes typo on base branch. Import typo preventing build.

File: src/components/__tests__/__snapshots__/Footer.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+  <div
+    className="block-center"
+  >
+    <div
+      className="footer-text"
```
Comment (rainrivas) [inline src/components/__tests__/__snapshots__/Footer.test.js.snap:43]: @jwu910 I personally prefer this to be on the same line as the icons. E.G. https://gyazo.com/53de944ab4fd43ea5508351473e35ee4
```
<table><tbody><tr><td><a href="https://github.com/5-gwoap" class="social-icon"><i class="fa fa-github fa-2x"></i></a></td><td><div class="footer-text">Code written under <a href="https://opensource.org/licenses/mit-license.html">MIT license</a></div></td><td><a href="mailto:5gwoap@gmail.com" class="social-icon"><i class="fa fa-envelope fa-2x"></i></a></td></tr></tbody></table>
```
> This just moves the license to the middle. Other options are to move it to either end, but this is just personal pref, no deal breaker

File: src/components/__tests__/__snapshots__/Main.test.js.snap [hunk 2]
```diff
@@ -42,5 +42,6 @@ exports[`Main Main snapshot 1`] = `
     />
      
   </Switch>
+  <Footer />
```
Comment (rainrivas) [inline src/components/__tests__/__snapshots__/Main.test.js.snap:45]: ~~Should this whitespace be here?~~
These are self closing, no worries

File: src/components/Main.js [hunk 3]
```diff
@@ -24,6 +25,8 @@ const Main = (props) => {
         <Route path="/secure/messaging" component={Messaging} /> {/* Secure */}
         <Route path="/secure/org" component={SecureOrg} /> {/* Secure */}
       </Switch>
+
+      <Footer />
```
Comment (rainrivas) [inline src/components/Main.js:29]: ~~Should we have this whitespace on closing tags?~~
These are self closing

**Review by rainrivas — COMMENTED** [review]
Looks like frontend branch has not been pulled in so the bugs that @psyduc fixed are missing but should be there once we rebase this onto the destination

**Review by rainrivas — APPROVED** [review]

**Review by phuchle — CHANGES_REQUESTED** [review]
Overall code is good we may want to use more semantic HTML5 tags like `<header />` and `<footer />`.
Further reading on that:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Content_sectioning

On Ubuntu with Chrome, I'm seeing overflow on the x axis on the landing page.  Also, I'm still seeing the nav issue that duc's branch should have fixed.  Did you merge from frontend-content branch into this one yet?

Josh edit: code styling

**Comment by jwu910** [comment]
@phuchle I think this was opened close to when @psyduc had his last changes merged and but i branched out and sent he PR before we had a chance to pull in changes form the base branch frontend-content.

It currently doesn't look like it has duc's latest commits in history

**Review by psyduc — APPROVED** [review]

**Review by phuchle — APPROVED** [review]

**Review by jwu910 — COMMENTED** [review]
LOL phuc i see what you mean now by using `<footer/>` and `<header/>`

**Pull request merged**